source/Keystone/ApplicationCredentials.rst

```diff
@@ -0,0 +1,179 @@
+=======================
+Application Credentials
+=======================
+
+Application credentials can be used to allow applications or other users certain access to specific projects
+without embedding the main user's password in the application configuration.
+
+Application credentials can be created either through the web interface or command line
+
+
+
+Create Credential from the Web interface
+########################################
+
+The option for generating application credentials can be found under *Identity -> Application Credentials*.
+Select *Create Application Credential*.
+
+The form has 7 fields:
+
+- **Name:** The name that is given to the application credential.
+
+- **Description:** Description of the application credential. This could be the description of which application will use this credential or the purpose of the credential.
+
+- **Secret:** A password for the credential which will be used in order to authenticate access through keystone. **This must be different to the user's password.**
```

Comment:
Does this need to be set? IIRC if unset it will autogenerate
The alternative is to steer people to `pwgen` (or similar) to generate a random password

Reply:
In the CLI it can be left unset and a secret is generated randomly. Let me check on the web interface quickly.

Reply:
Yeah it will autogenerate a secret